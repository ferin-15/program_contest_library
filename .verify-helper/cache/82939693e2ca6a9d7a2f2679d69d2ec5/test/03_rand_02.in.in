30 3
7 15 15 8 9 15 6 1 9 2 2 6 12 4 3 6 2 2 1 1 15 13 12 6 2 10 14 3 10 1
