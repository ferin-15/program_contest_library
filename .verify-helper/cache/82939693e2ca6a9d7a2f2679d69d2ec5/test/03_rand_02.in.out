7 8 8 8 6 1 1 1 2 2 2 4 3 3 2 2 1 1 1 1 12 6 2 2 2 3 3 1
